app/controllers/admin/deduplicate_patients_controller.rb

```diff
@@ -3,15 +3,15 @@ class Admin::DeduplicatePatientsController < AdminController
   DUPLICATE_LIMIT = 250
 
   def show
-    authorize { current_admin.power_user? }
+    authorize { current_admin.current_admin.accessible_organizations(:manage).any? }
```

Comment:
Did you mean to duplicate `current_admin` here?

Reply:
Oh thanks 👾